### PR TITLE
use python-decouple to get .env

### PR DESCRIPTION
**Pull request by silkemarie**
integrates python-decouple

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 database.py
```
Comment (shahnozaAI) [inline .gitignore:1]: do we still need to gitignore it? :)

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
 config.txt
+settings.py
```
Comment (shahnozaAI) [inline .gitignore:3]: what is this file for? :black_cat:

File: api/database.py [hunk 3]
```diff
@@ -1,11 +1,10 @@
 from sqlalchemy import create_engine
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
+from decouple import config
 
 
-#SQLALCHEMY_DATABASE_URL = "sqlite:///./sql_api.db"
-SQLALCHEMY_DATABASE_URL = "postgresql://postgres:postgres@localhost:5432/hello"
-#SQLALCHEMY_DATABASE_URL = f"postgresql://{settings.POSTGRES_USER}:{settings.POSTGRES_PASSWORD}@{settings.DATABASE_HOSTNAME}:{settings.DATABASE_PORT}/hello"
+SQLALCHEMY_DATABASE_URL = config("DATABASE_URL")
```
Comment (shahnozaAI) [inline api/database.py:7]: :clap: :clap: :clap:

File: .gitignore (hunk 1 above)
Comment (silkemarie) [reply .gitignore:1]: that's true!

File: .gitignore (hunk 2 above)
Comment (silkemarie) [reply .gitignore:3]: It was for decouple stuff, but I since moved it 😅  it's deleted now!

**Pull request merged**